pkg/controller/metadata/metadata_controller.go

```diff
@@ -520,11 +493,9 @@ func (r *ReconcileMetadata) Reconcile(request reconcile.Request) (reconcile.Resu
 		if err != nil {
 			return reconcile.Result{}, fmt.Errorf("failed to update Secret %s: %s", foundSecret.ObjectMeta.Name, err)
 		}
-		log.Info("updated-secret",
-			"namespace", foundSecret.Namespace,
-			"name", foundSecret.Name,
-			"version", currentVersion,
-		)
+		logInfo("Updated metadata secret", foundSecret.ObjectMeta, "version", currentVersion)
+	} else {
+		logInfo("Metadata up-to-date, not regenerating at this time", instance.ObjectMeta)
```

Comment:
This could get logged very often as k8s checks for reconciliation, do we want to log this no-op path?

Reply:
We log the begin message every time so I guess that's fine

Reply:
```
{
  "level": "info",
  "ts": "2019-09-02T13:20:50Z",
  "logger": "controller",
  "msg": "Beginning Reconcile for metadata",
  "namespace": "sandbox-proxy-node-dev",
  "name": "test-proxy-node-metadata"
}
```